### DISCUSSION TITLE
Useful Links

### DISCUSSION THREAD

**Discussion by sammou00** [hyf-Group2-fp/Just4Giving#11] [General]
You can post here the links you are using
===============================

UX/UI 
Front-End
Back-end-
DataBase
Dev-OPs

**Comment by sammou00** [comment]
Ux/UI [Free Logo Deisgn](https://editor.freelogodesign.org/en/logo/edit/c028f16b766d41f28e2ea9791cf9cf8c?category=0)

**Comment by AnisyaPurnama** [comment]
Could be useful for front-end : 

- https://stackoverflow.com/questions/38792005/how-to-change-the-bootstrap-primary-color
- https://www.youtube.com/watch?v=bBSrMEo90uA

**Comment by DIVYASREE345** [comment]
For the landing page - [React bootstrap landing page](https://www.youtube.com/playlist?list=PLup_wjXHoCRiwTbG7BdeplQZKv_kf0aYZ)
For React studies - [Tutorial](https://www.youtube.com/watch?v=TRCDsB9i3bI)

**Comment by DIVYASREE345** [comment]
A VERY IMPORTANT VIDEO FOR OUR PROJECT[ E-commerce](https://www.youtube.com/watch?v=TRCDsB9i3bI)

**Comment by sammou00** [comment]
Daily standup 5 / 05

1- light weight landing page .

2- Add some text 

3- video is optional 

4- text information explain our project

**Comment by bermarte** [comment]
### Some links to projects about immigration:
HYF Cass 6 - Language app https://github.com/akbelcolak/language-platform
Urban.Brussels https://github.com/oSoc20/urban-brussels
Apps for migrants https://www.infomigrants.net/en/post/2287/a-list-of-apps-for-refugees
Refugee buddy https://apps.apple.com/us/app/refugee-buddy/id1331976412
I'd like to expand the idea of Dirk
USA hello https://find.usahello.org/#/welcome

**Comment by bermarte** [comment]
### categories (initial list):

Free food
Home & garden
Tools
Furniture
Pets
Baby
Fashion
Sport
Tech / electronics
Books
Other

**Reply by sammou00** [reply]
Redux thunk : https://daveceddia.com/what-is-a-thunk/
